src/github/nag.rs

```diff
@@ -185,7 +130,147 @@ fn update_proposal_review_status(proposal_id: i32) -> DashResult<()> {
     Ok(())
 }
 
+fn update_poll_response_status(poll_id: i32) -> DashResult<()> {
+    let conn = &*DB_POOL.get()?;
+    // this is an updated comment from the bot itself
+
+    // parse out each "responded" status for each user, then update them
+
+    let survey: Poll = poll::table.find(poll_id).first(conn)?;
+
+    // don't update any statuses if the poll is closed
+    if survey.poll_closed {
+        return Ok(());
+    }
+
+    let comment: IssueComment = issuecomment::table
+        .find(survey.fk_bot_tracking_comment)
+        .first(conn)?;
+
+    // parse the status comment and mark any new responses as responded
+    for username in parse_ticky_boxes("poll", survey.id, &comment) {
+        let user: GitHubUser = githubuser::table
+            .filter(githubuser::login.eq(username))
+            .first(conn)?;
+
+        {
+            use domain::schema::poll_response_request::dsl::*;
+            let mut review_request: PollResponseRequest = poll_response_request
+                .filter(fk_poll.eq(survey.id))
+                .filter(fk_respondent.eq(user.id))
+                .first(conn)?;
+
+            review_request.responded = true;
+            diesel::update(poll_response_request.find(review_request.id))
+                .set(&review_request)
+                .execute(conn)?;
+        }
+    }
+
+    Ok(())
+}
+
+fn parse_ticky_boxes<'a>(what: &'a str, id: i32, comment: &'a IssueComment)
+    -> impl Iterator<Item = &'a str>
+{
+    comment.body.lines().filter_map(move |line| if line.starts_with("* [") {
+        let l = line.trim_left_matches("* [");
+        let reviewed = l.starts_with('x');
+        let remaining = l.trim_left_matches("x] @").trim_left_matches(" ] @");
+
+        if let Some(username) = remaining.split_whitespace().next() {
+            trace!("reviewer parsed as reviewed? {} (line: \"{}\")",
+                    reviewed, l);
+
+            if reviewed { Some(username) } else { None }
+        } else {
+            warn!("An empty usename showed up in comment {} for {} {}",
+                  comment.id, what, id);
+            None
+        }
+    } else {
+        None
+    })
+}
+
 fn evaluate_nags() -> DashResult<()> {
+    evaluate_pendings()?;
```

Comment:
Looking at this now, should we attempt to make progress on all three and combine the errors if more than one fails?